[2017-09-16 23:37:50,57] [info] Running with database db.url = jdbc:mysql://localhost:2801/cromwell_db?useSSL=false&rewriteBatchedStatements=true
[2017-09-16 23:38:00,77] [info] Running with database db.url = jdbc:mysql://localhost:2801/cromwell_db?useSSL=false&rewriteBatchedStatements=true
[2017-09-16 23:38:01,24] [info] Slf4jLogger started
[2017-09-16 23:38:01,44] [info] Metadata summary refreshing every 2 seconds.
[2017-09-16 23:38:01,46] [info] WriteMetadataActor configured to write to the database with batch size 200 and flush rate 5 seconds.
[2017-09-16 23:38:02,91] [info] CallCacheWriteActor configured to write to the database with batch size 100 and flush rate 3 seconds.
[2017-09-16 23:38:03,73] [info] SingleWorkflowRunnerActor: Submitting workflow
[2017-09-16 23:38:03,77] [info] Workflow c44b0020-ae84-49f0-a66d-f464faa48734 submitted.
[2017-09-16 23:38:03,77] [info] SingleWorkflowRunnerActor: Workflow submitted [38;5;2mc44b0020-ae84-49f0-a66d-f464faa48734[0m
[2017-09-16 23:38:03,78] [info] 1 new workflows fetched
[2017-09-16 23:38:03,78] [info] WorkflowManagerActor Starting workflow [38;5;2mc44b0020-ae84-49f0-a66d-f464faa48734[0m
[2017-09-16 23:38:03,78] [info] WorkflowManagerActor Successfully started WorkflowActor-c44b0020-ae84-49f0-a66d-f464faa48734
[2017-09-16 23:38:03,78] [info] Retrieved 1 workflows from the WorkflowStoreActor
[2017-09-16 23:38:04,87] [info] MaterializeWorkflowDescriptorActor [[38;5;2mc44b0020[0m]: Call-to-Backend assignments: PairedEndSingleSampleWorkflow.ConvertToCram -> SGE, PairedEndSingleSampleWorkflow.CollectUnsortedReadgroupBamQualityMetrics -> SGE, PairedEndSingleSampleWorkflow.CheckContamination -> SGE, PairedEndSingleSampleWorkflow.HaplotypeCaller -> SGE, PairedEndSingleSampleWorkflow.SortAndFixReadGroupBam -> SGE, PairedEndSingleSampleWorkflow.ValidateBamFromCram -> SGE, PairedEndSingleSampleWorkflow.CollectReadgroupBamQualityMetrics -> SGE, PairedEndSingleSampleWorkflow.CollectGvcfCallingMetrics -> SGE, PairedEndSingleSampleWorkflow.MergeVCFs -> SGE, PairedEndSingleSampleWorkflow.GatherBqsrReports -> SGE, PairedEndSingleSampleWorkflow.CollectWgsMetrics -> SGE, PairedEndSingleSampleWorkflow.CreateSequenceGroupingTSV -> SGE, PairedEndSingleSampleWorkflow.SamToFastqAndBwaMem -> SGE, PairedEndSingleSampleWorkflow.GatherBamFiles -> SGE, PairedEndSingleSampleWorkflow.BaseRecalibrator -> SGE, PairedEndSingleSampleWorkflow.SortAndFixSampleBam -> SGE, PairedEndSingleSampleWorkflow.CramToBam -> SGE, PairedEndSingleSampleWorkflow.CheckFingerprint -> SGE, PairedEndSingleSampleWorkflow.ValidateAggregatedSamFile -> SGE, PairedEndSingleSampleWorkflow.CollectRawWgsMetrics -> SGE, PairedEndSingleSampleWorkflow.GetBwaVersion -> SGE, PairedEndSingleSampleWorkflow.CrossCheckFingerprints -> SGE, PairedEndSingleSampleWorkflow.ValidateReadGroupSamFile -> SGE, PairedEndSingleSampleWorkflow.CollectAggregationMetrics -> SGE, PairedEndSingleSampleWorkflow.MarkDuplicates -> SGE, PairedEndSingleSampleWorkflow.CalculateReadGroupChecksum -> SGE, PairedEndSingleSampleWorkflow.CollectQualityYieldMetrics -> SGE, PairedEndSingleSampleWorkflow.ValidateGVCF -> SGE, PairedEndSingleSampleWorkflow.MergeBamAlignment -> SGE, PairedEndSingleSampleWorkflow.CheckFinalVcfExtension -> SGE, PairedEndSingleSampleWorkflow.ApplyBQSR -> SGE
[2017-09-16 23:38:07,17] [info] WorkflowExecutionActor-c44b0020-ae84-49f0-a66d-f464faa48734 [[38;5;2mc44b0020[0m]: Starting calls: PairedEndSingleSampleWorkflow.CheckFinalVcfExtension:NA:1, PairedEndSingleSampleWorkflow.CreateSequenceGroupingTSV:NA:1, PairedEndSingleSampleWorkflow.GetBwaVersion:NA:1
[2017-09-16 23:38:09,70] [info] WorkflowExecutionActor-c44b0020-ae84-49f0-a66d-f464faa48734 [[38;5;2mc44b0020[0m]: Starting calls: PairedEndSingleSampleWorkflow.CollectQualityYieldMetrics:0:1, PairedEndSingleSampleWorkflow.SamToFastqAndBwaMem:0:1
[2017-09-16 23:38:21,21] [info] DispatchedConfigAsyncJobExecutionActor [[38;5;2mc44b0020[0mPairedEndSingleSampleWorkflow.SamToFastqAndBwaMem:0:1]: [38;5;5mset -o pipefail
set -e
# set the bash variable needed for the command-line
bash_ref_fasta=/home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/inputs/home/projects/dp_00005/data/cromwell_test/genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta
# if ref_alt has data in it, we proceed
if [ -s /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/inputs/home/projects/dp_00005/data/cromwell_test/genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.fasta.64.alt ]; then
  java -Xmx3000m \
    -jar /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/inputs/home/projects/dp_00005/apps/src/picard/build/libs/picard-2.11.0-2-g2206c08-SNAPSHOT-all.jar \
    SamToFastq \
    INPUT=/home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/inputs/home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.bam \
    FASTQ=/dev/stdout \
    INTERLEAVE=true \
    NON_PF=true | \
  /services/tools/bwa/0.7.15/bwa mem -K 100000000 -p -v 3 -t 28 -Y $bash_ref_fasta /dev/stdin -  2> >(tee /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmerged.bwa.stderr.log >&2) | \
  /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/inputs/home/projects/dp_00005/apps/src/samtools-1.5/samtools view -1 - > /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmerged.bam
  grep -m1 "read .* ALT contigs" /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmerged.bwa.stderr.log | \
  grep -v "read 0 ALT contigs"
# else ref_alt is empty or could not be found, so we bail out
else
  exit 1;
fi[0m
[2017-09-16 23:38:21,23] [info] DispatchedConfigAsyncJobExecutionActor [[38;5;2mc44b0020[0mPairedEndSingleSampleWorkflow.SamToFastqAndBwaMem:0:1]: executing: qsub \
-V \
-N cromwell_c44b0020_SamToFastqAndBwaMem \
-d /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0 \
-o /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/execution/stdout \
-e /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/execution/stderr \
-W group_list=dp_00005 -A dp_00005 -l nodes=1:ppn=28,walltime=100:00:00 \
/home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-SamToFastqAndBwaMem/shard-0/execution/script
[2017-09-16 23:38:22,36] [info] DispatchedConfigAsyncJobExecutionActor [[38;5;2mc44b0020[0mPairedEndSingleSampleWorkflow.SamToFastqAndBwaMem:0:1]: job id: 18368127.risoe-r04-sn064.cm.cluster
[2017-09-16 23:38:22,37] [info] DispatchedConfigAsyncJobExecutionActor [[38;5;2mc44b0020[0mPairedEndSingleSampleWorkflow.SamToFastqAndBwaMem:0:1]: Status change from - to WaitingForReturnCodeFile
[2017-09-16 23:46:29,50] [[38;5;220mwarn[0m] Localization via hard link has failed: /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-CollectQualityYieldMetrics/shard-0/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics -> /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
[2017-09-16 23:46:29,51] [[38;5;220mwarn[0m] Couldn't find a suitable DSN, defaulting to a Noop one.
[2017-09-16 23:46:29,52] [info] Using noop to send events.
[2017-09-16 23:46:29,54] [[38;5;220mwarn[0m] Localization via copy has failed: /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
[2017-09-16 23:46:29,58] [[38;5;1merror[0m] Failed copying cache results for job PairedEndSingleSampleWorkflow.CollectQualityYieldMetrics:0:1, invalidating cache entry.
lenthall.exception.AggregatedException: :
Could not cache /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics -> /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-CollectQualityYieldMetrics/shard-0/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics:
	/home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics doesn't exists
	File not found /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-CollectQualityYieldMetrics/shard-0/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics -> /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
	File not found /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
	File not found /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
	at lenthall.util.TryUtil$.sequenceIterable(TryUtil.scala:26)
	at lenthall.util.TryUtil$.sequence(TryUtil.scala:33)
	at cromwell.backend.sfs.SharedFileSystemCacheHitCopyingActor.duplicate(SharedFileSystemCacheHitCopyingActor.scala:20)
	at cromwell.backend.standard.callcaching.StandardCacheHitCopyingActor$$anonfun$1.applyOrElse(StandardCacheHitCopyingActor.scala:144)
	at cromwell.backend.standard.callcaching.StandardCacheHitCopyingActor$$anonfun$1.applyOrElse(StandardCacheHitCopyingActor.scala:129)
	at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:34)
	at akka.actor.FSM.processEvent(FSM.scala:665)
	at akka.actor.FSM.processEvent$(FSM.scala:662)
	at cromwell.backend.standard.callcaching.StandardCacheHitCopyingActor.processEvent(StandardCacheHitCopyingActor.scala:110)
	at akka.actor.FSM.akka$actor$FSM$$processMsg(FSM.scala:659)
	at akka.actor.FSM$$anonfun$receive$1.applyOrElse(FSM.scala:653)
	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)
	at scala.PartialFunction$OrElse.applyOrElse(PartialFunction.scala:172)
	at akka.actor.Actor.aroundReceive(Actor.scala:514)
	at akka.actor.Actor.aroundReceive$(Actor.scala:512)
	at cromwell.backend.standard.callcaching.StandardCacheHitCopyingActor.aroundReceive(StandardCacheHitCopyingActor.scala:110)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:527)
	at akka.actor.ActorCell.invoke(ActorCell.scala:496)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
	at akka.dispatch.Mailbox.run(Mailbox.scala:224)
	at akka.dispatch.Mailbox.exec(Mailbox.scala:234)
	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
	Suppressed: lenthall.exception.AggregatedException: Could not cache /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics -> /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-CollectQualityYieldMetrics/shard-0/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics:
/home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics doesn't exists
File not found /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-CollectQualityYieldMetrics/shard-0/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics -> /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yield_metrics
File not found /home/projects/dp_00005/data/bgi_received_11092017/F17FTSEUHT0065_HUMdnpR/Raw/36PGF/170830_X512_FCH3N2TCCXY_L5_8521708000404.unmapped.quality_yie[2017-09-18 14:58:32,09] [[38;5;1merror[0m] Failed to summarize metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5010ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:58:32,09] [[38;5;1merror[0m] Error summarizing metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5010ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:58:44,12] [[38;5;1merror[0m] Failed to summarize metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5009ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:58:44,12] [[38;5;1merror[0m] Error summarizing metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5009ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:58:56,14] [[38;5;1merror[0m] Failed to summarize metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5001ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:58:56,14] [[38;5;1merror[0m] Error summarizing metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5001ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
[2017-09-18 14:59:08,15] [[38;5;1merror[0m] Failed to summarize metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5000ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor68.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:632)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1016)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2194)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2225)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
	at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 common frames omitted
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at com.mysql.jdbc.util.ReadAheadInputStream.fill(ReadAheadInputStream.java:101)
	at com.mysql.jdbc.util.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:144)
	at com.mysql.jdbc.util.ReadAheadInputStream.read(ReadAheadInputStream.java:174)
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:3008)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:567)
	... 24 common frames omitted
[2017-09-18 14:59:08,16] [[38;5;1merror[0m] Error summarizing metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5000ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor68.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:632)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1016)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2194)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2225)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
	at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 common frames omitted
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at com.mysql.jdbc.util.ReadAheadInputStream.fill(ReadAheadInputStream.java:101)
	at com.mysql.jdbc.util.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:144)
	at com.mysql.jdbc.util.ReadAheadInputStream.read(ReadAheadInputStream.java:174)
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:3008)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:567)
	... 24 common frames omitted
[2017-09-18 14:59:20,17] [[38;5;1merror[0m] Failed to summarize metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5000ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor68.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:632)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1016)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2194)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2225)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
	at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 common frames omitted
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at com.mysql.jdbc.util.ReadAheadInputStream.fill(ReadAheadInputStream.java:101)
	at com.mysql.jdbc.util.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:144)
	at com.mysql.jdbc.util.ReadAheadInputStream.read(ReadAheadInputStream.java:174)
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:3008)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:567)
	... 24 common frames omitted
[2017-09-18 14:59:20,18] [[38;5;1merror[0m] Error summarizing metadata
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5000ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor68.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:632)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1016)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2194)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2225)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
	at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 common frames omitted
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at com.mysql.jdbc.util.ReadAheadInputStream.fill(ReadAheadInputStream.java:101)
	at com.mysql.jdbc.util.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:144)
	at com.mysql.jdbc.util.ReadAheadInputStream.read(ReadAheadInputStream.java:174)
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:3008)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:567)
	... 24 common frames omitted
[2017-09-18 14:59:27,17] [[38;5;1merror[0m] SingleWorkflowRunnerActor received Failure message: db - Connection is not available, request timed out after 5000ms.
java.sql.SQLTransientConnectionException: db - Connection is not available, request timed out after 5000ms.
	at com.zaxxer.hikari.pool.HikariPool.createTimeoutException(HikariPool.java:548)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:186)
	at com.zaxxer.hikari.pool.HikariPool.getConnection(HikariPool.java:145)
	at com.zaxxer.hikari.HikariDataSource.getConnection(HikariDataSource.java:83)
	at slick.jdbc.hikaricp.HikariCPJdbcDataSource.createConnection(HikariCPJdbcDataSource.scala:18)
	at slick.jdbc.JdbcBackend$BaseSession.<init>(JdbcBackend.scala:439)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:47)
	at slick.jdbc.JdbcBackend$DatabaseDef.createSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession(BasicBackend.scala:218)
	at slick.basic.BasicBackend$DatabaseDef.acquireSession$(BasicBackend.scala:217)
	at slick.jdbc.JdbcBackend$DatabaseDef.acquireSession(JdbcBackend.scala:38)
	at slick.basic.BasicBackend$DatabaseDef$$anon$2.run(BasicBackend.scala:239)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.GeneratedConstructorAccessor68.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:989)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:632)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1016)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2194)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2225)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2024)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:779)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.GeneratedConstructorAccessor5.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:389)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:95)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:101)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
	at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	... 3 common frames omitted
Caused by: java.net.SocketTimeoutException: Read timed out
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:171)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at com.mysql.jdbc.util.ReadAheadInputStream.fill(ReadAheadInputStream.java:101)
	at com.mysql.jdbc.util.ReadAheadInputStream.readFromUnderlyingStreamIfNecessary(ReadAheadInputStream.java:144)
	at com.mysql.jdbc.util.ReadAheadInputStream.read(ReadAheadInputStream.java:174)
	at com.mysql.jdbc.MysqlIO.readFully(MysqlIO.java:3008)
	at com.mysql.jdbc.MysqlIO.readPacket(MysqlIO.java:567)
	... 24 common frames omitted
[2017-09-18 14:59:27,18] [info] Message [cromwell.core.actor.StreamActorHelper$StreamFailed] without sender to Actor[akka://cromwell-system/deadLetters] was not delivered. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[2017-09-18 14:59:27,18] [info] Message [cromwell.core.actor.StreamActorHelper$StreamFailed] without sender to Actor[akka://cromwell-system/deadLetters] was not delivered. [2] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
db - Connection is not available, request timed out after 5000ms.
[2017-09-18 14:59:27,21] [info] Automatic shutdown of the async connection
[2017-09-18 14:59:27,21] [info] Gracefully shutdown sentry threads.
[2017-09-18 14:59:27,21] [info] Shutdown finished.
nt of stderr for potential additional information: /home/projects/dp_00005/data/cromwell_test/wdl/scripts/CBS_pipelines_realdata_STAGING/cromwell-executions/PairedEndSingleSampleWorkflow/c44b0020-ae84-49f0-a66d-f464faa48734/call-ValidateReadGroupSamFile/shard-0/execution/stderr
[2017-09-18 01:13:14,78] [info] WorkflowManagerActor WorkflowActor-c44b0020-ae84-49f0-a66d-f464faa48734 is in a terminal state: WorkflowFailedState
[2017-09-18 01:14:09,03] [info] SingleWorkflowRunnerActor workflow finished with status 'Failed'.
[2017-09-18 01:14:09,05] [info] Message [cromwell.core.actor.StreamActorHelper$StreamFailed] without sender to Actor[akka://cromwell-system/deadLetters] was not delivered. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
[2017-09-18 01:14:09,05] [info] Message [cromwell.core.actor.StreamActorHelper$StreamFailed] without sender to Actor[akka://cromwell-system/deadLetters] was not delivered. [2] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
Workflow c44b0020-ae84-49f0-a66d-f464faa48734 transitioned to state Failed
[2017-09-18 01:14:09,09] [info] Automatic shutdown of the async connection
[2017-09-18 01:14:09,09] [info] Gracefully shutdown sentry threads.
[2017-09-18 01:14:09,09] [info] Shutdown finished.
